### PR TITLE
Auto enable Litestar integration

### PR DESCRIPTION
**Pull request by provinzkraut**
Auto enable the Litestar integration added in #3358.

---

Thank you for contributing to `sentry-python`! Please add tests to validate your changes, and lint your code using `tox -e linters`.

Running the test suite on your PR might require maintainer approval. The AWS Lambda tests additionally require a maintainer to add a special label, and they will fail until this label is added.

### CODE REVIEW

**Comment by sentrivana** [comment]
Hey @provinzkraut, thanks for the PR -- we can merge this once the Litestar integration has been available for some time. I'd wait for couple more releases. We also have a new major coming up soonish (probably still this year) so if we don't enable the integration until then, SDK 3.0 would be a good point to do it.

**Review by antonpirker — APPROVED** [review]
Some time has passed and there were no bug reports about the Litestar integration, so we can auto enable it.

**Pull request merged**
